docs/concepts/models.md

```diff
@@ -1222,8 +1222,8 @@ values of instance attributes will raise errors. See the [API reference][pydanti
     This config flag is deprecated in Pydantic V2, and has been replaced with `frozen`.
 
 !!! warning
-    Immutability in Python is never strict. If developers are determined/stupid they can always
-    modify a so-called "immutable" object.
+    In Python, strict immutability is not enforced. Developers have the ability to modify objects
```

Comment:
```suggestion
    Immutability in Python is never strict. Developers have the ability to modify objects
```

Reply:
I like the `is not enforced` language better. I find it semantically clearer than `strict`.

Reply:
That sounds better to me